src/components/ProjectSettingsTabs/PluginsTab/getFormSchema.ts

```diff
@@ -0,0 +1,182 @@
+import { Field } from "@rjsf/core";
+import { SpruceFormProps } from "components/SpruceForm";
+import { CardFieldTemplate } from "components/SpruceForm/FieldTemplates";
+import widgets from "components/SpruceForm/Widgets";
+import { placeholderIf, radioBoxOptions } from "../utils";
+
+export const getFormSchema = (
+  useRepoSettings: boolean,
+  repoData?: any,
+  formData?: any
+): {
+  fields: Record<string, Field>;
+  schema: SpruceFormProps["schema"];
+  uiSchema: SpruceFormProps["uiSchema"];
+} => ({
+  fields: {},
+  schema: {
+    type: "object" as "object",
+    properties: {
+      performanceSettings: {
+        type: "object" as "object",
+        title: "Performance Plugins",
+        properties: {
+          perfEnabled: {
+            type: ["boolean", "null"],
+            title: "Build-break notifications",
```

Comment:
[nit] I've been capitalizing the headings for these forms
```suggestion
            title: "Build-break Notifications",
```